draft-ietf-quic-multipath.md

```diff
@@ -667,23 +759,23 @@ the packet number alone would not guarantee the uniqueness of the nonce.
 
 In order to guarantee the uniqueness of the nonce, the nonce N is
 calculated by combining the packet protection IV with the packet number
-and with the least significant 32 bits of the Destination Connection ID
-sequence number.
+and with the least significant 32 bits of the path identifier pre-allocated 
+for the Destination Connection ID.
 
-{{Section 19 of QUIC-TRANSPORT}} encodes the Connection ID Sequence
-Number as a variable-length integer,
-allowing values up to 2^62-1; in this specification, a range of less than 2^32-1
+{{mp-new-conn-id-frame}} encodes the Path Identifier for Connection IDs 
+as a variable-length integer, allowing values up to 2^62-1; 
+in this specification, a range of less than 2^32-1
 values MUST be used before updating the packet protection key.
 
 To calculate the nonce, a 96 bit path-and-packet-number is composed of the least
-significant 32 bits of the Connection ID Sequence Number in network byte order,
+significant 32 bits of the Path Identifier in network byte order,
 two zero bits, and the 62 bits of the reconstructed QUIC packet number in
 network byte order. If the IV is larger than 96 bits, the path-and-packet-number
 is left-padded with zeros to the size of the IV. The exclusive OR of the padded
 packet number and the IV forms the AEAD nonce.
 
```

Comment:
See comments above. In my mind, using the path ID in the nonce pretty much mandates unique path ID. Something like a monotonously increasing number.